SharpPcap/LibPcap/PcapInterface.cs

```diff
@@ -184,5 +204,32 @@ public override string ToString()
             sb.AppendFormat("Flags: {0}\n", Flags);
             return sb.ToString();
         }
+
+        static internal PcapInterface[] GetAllPcapInterfaces(IntPtr devicePtr)
```

Comment:
Thoughts on returning the List<PcapInterface> from this routine vs. converting it to an array?

Reply:
Originally, I wanted the method to have a signature similar to `GetAllNetworkInterfaces`.
But since it's internal not public that logic is no longer applicable